### PR TITLE
Add case insensitive filters to content manager

### PR DESCRIPTION
**Pull request by derrickmehaffy**
TODO:

- [x] Add eqi
- [ ] Add nei
- [x] Add containsi
- [x] Add notContainsi
- [x] Add startsWithi
- [x] Add endsWithi
- [ ] Handle translations

Translations:

- [ ] ar
- [ ] ca
- [ ] cs
- [ ] de
- [ ] dk
- [x] en
- [ ] es
- [ ] eu
- [ ] fr
- [ ] gu
- [ ] he
- [ ] hi
- [ ] hu
- [ ] id
- [ ] it
- [ ] ja
- [ ] ko
- [ ] ml
- [ ] ms
- [ ] nl
- [ ] no
- [ ] pl
- [ ] pt-BR
- [ ] pt
- [ ] ru
- [ ] sa
- [ ] sk
- [ ] sv
- [ ] th
- [ ] tr
- [ ] uk
- [ ] vi
- [ ] zh-Hans
- [ ] zh

### CODE REVIEW

File: packages/core/admin/admin/src/translations/en.json [hunk 1]
```diff
@@ -529,18 +529,23 @@
   "components.AutoReloadBlocker.description": "Run Strapi with one of the following commands:",
   "components.AutoReloadBlocker.header": "Reload feature is required for this plugin.",
   "components.ErrorBoundary.title": "Something went wrong...",
-  "components.FilterOptions.FILTER_TYPES.$contains": "contains (case sensitive)",
+  "components.FilterOptions.FILTER_TYPES.$contains": "contains",
+  "components.FilterOptions.FILTER_TYPES.$containsi": "contains (case insensitive)",
   "components.FilterOptions.FILTER_TYPES.$endsWith": "ends with",
+  "components.FilterOptions.FILTER_TYPES.$endsWithi": "ends with",
```
Comment (marob) [inline packages/core/admin/admin/src/translations/en.json:535]:
```suggestion
  "components.FilterOptions.FILTER_TYPES.$endsWithi": "ends with (case insensitive)",
```

File: packages/core/admin/admin/src/translations/ja.json [hunk 2]
```diff
@@ -324,18 +324,23 @@
   "components.AutoReloadBlocker.description": "Run Strapi with one of the following commands:",
   "components.AutoReloadBlocker.header": "プラグインを有効化するにはリロードが必要です",
   "components.ErrorBoundary.title": "なにかが間違っています...",
-  "components.FilterOptions.FILTER_TYPES.$contains": "contains (case sensitive)",
+  "components.FilterOptions.FILTER_TYPES.$contains": "contains",
+  "components.FilterOptions.FILTER_TYPES.$containsi": "contains (case insensitive)",
   "components.FilterOptions.FILTER_TYPES.$endsWith": "ends with",
+  "components.FilterOptions.FILTER_TYPES.$endsWithi": "ends with",
```
Comment (marob) [inline packages/core/admin/admin/src/translations/ja.json:330]:
```suggestion
  "components.FilterOptions.FILTER_TYPES.$endsWithi": "ends with (case insensitive)",
```

File: packages/core/admin/admin/src/translations/ko.json [hunk 3]
```diff
@@ -323,18 +323,23 @@
   "components.AutoReloadBlocker.description": "다음 명령어 중 하나를 사용하여 Strapi를 실행합니다:",
   "components.AutoReloadBlocker.header": "이 플러그인은 리로드 기능이 필요합니다.",
   "components.ErrorBoundary.title": "에러가 발생했습니다.",
-  "components.FilterOptions.FILTER_TYPES.$contains": "contains (대소문자 구분)",
+  "components.FilterOptions.FILTER_TYPES.$contains": "contains",
+  "components.FilterOptions.FILTER_TYPES.$containsi": "contains (case insensitive)",
   "components.FilterOptions.FILTER_TYPES.$endsWith": "ends with",
+  "components.FilterOptions.FILTER_TYPES.$endsWithi": "ends with",
```
Comment (marob) [inline packages/core/admin/admin/src/translations/ko.json:329]:
```suggestion
  "components.FilterOptions.FILTER_TYPES.$endsWithi": "ends with (case insensitive)",
```

**Comment by gu-stav** [comment]
I'll remove myself as assignee as long as this is in progress. Feel free to ping me again once it is in shape :)

**Comment by gu-stav** [comment]
@marob Right now it would be @derrickmehaffy. I think if you'd like to take it over and complete it, he would appreciate it. If you have any questions or need a review feel free to ping me.

**Comment by derrickmehaffy** [comment]
I don't really have the time to finish this and I'm wayyyyy outside my element getting this to work properly. If someone wants to take over it let me know else I'll close it.

**Comment by marob** [comment]
@derrickmehaffy @gu-stav I've continued (and finished) the MR in this PR: https://github.com/strapi/strapi/pull/16960
I did my best on the translation part.
Could you have a look?

**Comment by derrickmehaffy** [comment]
closed in favor of https://github.com/strapi/strapi/pull/16960

**Pull request closed without merging**